src/index.js

```diff
@@ -49,20 +50,131 @@ function addGuestDataToDOM() {
   guestId = parseInt(localStorage.getItem('guestId'));
   showBookings();
   $('#total-spent').text(`$${hotel.returnTotalSpent(guestId)}`);
-  $('#reward-remainder').text(`$${7000 - hotel.returnTotalSpent(guestId)}`)
+  $('#reward-remainder').text(`$${10000 - hotel.returnTotalSpent(guestId)}`)
 }
 
 function showBookings() {
+  $('#guest-booking-info').text('');
   let bookingDates = bookingsRepository.viewBookings(guestId).map(booking => booking.date)
   let sortedBookings = sortDates(bookingDates);
   sortedBookings.forEach(date => {
     $('#guest-booking-info').append(
-    `<b>DATE</b>: ${date}
-    <br/>
-    <br/>`)
+      `
+      <b>DATE</b>: ${date}
+      <br/>
+      <br/>
+      `
+    )
   })
 }
 
+$('#find-available-rooms-button').click(() => {
+  let numDate = numifyDate($('#start-date').val(), '-');
+  let date = stringifyDate(numDate);
+  if (checkDate(date)) {
+    showAvailableRooms(hotel.viewRoomsAvailable(date));
+  } else {
+    $('#available-rooms').text('');
+    $('#available-rooms').append(`
+      <br/>
+      That date is in the past.
+      <br/>
+      We do not have time travelling capabilities (yet).
+      <br/>
+      Please choose another date.
+      `
+    )
+  }
+});
+
+function showAvailableRooms(method) {
+  $('#available-rooms').text('');
+  if (method.length) {
+    method.forEach(room => {
+      $('#available-rooms').append(
+        `
+        <div class='individual-rooms'>
+        <b id='x${room.number}x'></b>
+        <b>ROOM TYPE</b>: ${room.roomType}
+        </br>
+        <b>NUMBER OF BEDS</b>: ${room.numBeds}
+        </br>
+        <b>BED SIZE</b>: ${room.bedSize}
+        </br>
+        <b>PRICE</b>: $${room.costPerNight}
+        <br/>
+        <br/>
+        </div>
+        `
+      )
+    })
+    $('.individual-rooms').addClass('hoverable');
+  } else {
+    $('#available-rooms').append(
+      `
+      <b>We are so sorry, but there are no rooms available for that date. Please try another date.</b>
+      `
+    )
+  }
+};
+
+$('#available-rooms').click(() => {
+  if ($('.individual-rooms').hasClass('hoverable')) {
+    $('.individual-rooms').removeClass('hoverable');
+    $(event.target).closest('div').addClass('clicked');
+  } else if ($(event.target).closest('div').hasClass('clicked')) {
+    $('.individual-rooms').addClass('hoverable');
+    $(event.target).closest('div').removeClass('clicked');
+  }
+});
+
+$('.complete-booking-button').click(() => {
+  if ($('.clicked').html()) {
+    completeBooking();
+  } else {
+    //throw error
+  }
+});
+
+function completeBooking() {
+  let numDate = numifyDate($('#start-date').val(), '-');
+  let bookingDate = stringifyDate(numDate);
```

Comment:
I saw that you created a function that would stringify the date or something, I think I made a comment about how you could have used JSON.stringify on that.